.github/workflows/deployment_vars.yml

```diff
@@ -0,0 +1,79 @@
+name: Deployment variable tests
+on:
+    pull_request:
+        paths:
+            - 'src/init/register_configure_agent.sh'
+            - '/home/dfolch/wazuh/src/win32/InstallerScripts.vbs'
```

Comment:
This should be `src/win32/InstallerScripts.vbs`?

Reply:
Done [d677dd2](https://github.com/wazuh/wazuh/pull/15038/commits/d677dd294ab0fc858d2f904fa9e68a8b555bf68c)